controllers/dopplersecret_controller.go

```diff
@@ -76,24 +78,29 @@ func (r *DopplerSecretReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}, nil
 	}
 
+	authNamespace := dopplerSecret.Spec.TokenSecretRef.Namespace
+	authName := dopplerSecret.Spec.TokenSecretRef.Name
+
 	// If omitted, the default namespace for references is the DopplerSecret's namespace
-	tokenSecretRefNamespace := dopplerSecret.Spec.TokenSecretRef.Namespace
-	if tokenSecretRefNamespace == "" {
-		tokenSecretRefNamespace = dopplerSecret.Namespace
+	if authNamespace == "" {
+		authNamespace = dopplerSecret.Namespace
 	}
-	managedSecretRefNamespace := dopplerSecret.Spec.ManagedSecretRef.Namespace
-	if managedSecretRefNamespace == "" {
-		managedSecretRefNamespace = dopplerSecret.Namespace
+	managedNamespace := dopplerSecret.Spec.ManagedSecretRef.Namespace
```

Comment:
nit/optional: `managedSecretNamespace` might be a better name for this, `managedNamespace` doesn't really describe what it is.